api/decorators/index.ts

```diff
@@ -0,0 +1,46 @@
+export function IsNotEmpty(
+    target: any,
+    methodName: string,
+    parameterIndex: number
+) {
+    if (!target?.validations) {
+        target.validations = [];
+    }
+
+    target.validations.push(`${methodName}:${parameterIndex}:${IsNotEmpty.name}`);
+}
```

Comment:
Registers validation for argument (it's not possible to get value of the argument in an argument decorator which is weird)